MyException :: MyException(std :: string errm) : errorMessage(errm){}

std :: string MyException :: get_errorMessage() {return errorMessage;}
